docs/api/wrappers/shimmy_wrappers.md

```diff
@@ -25,8 +25,9 @@ The [Shimmy](https://shimmy.farama.org/) package (`pip install shimmy`) allows c
 
 To load a DeepMind Control [multi-agent soccer game](https://github.com/deepmind/dm_control/blob/main/dm_control/locomotion/soccer/README.md):
 
-```python
+```python notest
 from shimmy import DmControlMultiAgentCompatibilityV0
+#TODO ModuleNotFoundError: No module named 'dm_control'
```

Comment:
This one requires pip install shimmy[dm-control] or all, I think it’s possible to make a pytest fixture or some sort of thing to basically say when doing import shimmy, we have to install it (or mock it but ideally it would actually test it)

Reply:
I’d have to test locally but my guess is if the pytest docs can run the pettingzoo scripts then we can have it install shimmy as well.